native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -157,6 +169,22 @@ const BpkBadge = (props: Props) => {
 
   return (
     <View style={viewStyle}>
+      {icons &&
+        icons.map((i, n) => (
```

Comment:
One-letter variable names can be a bit obtuse - can we use `iconName` and `index` instead?

Reply:
I agree - I hate it when people write code like this. Goddamnit!